osquery/core/darwin/iokit.hpp

```diff
@@ -0,0 +1,56 @@
+/**
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under both the Apache 2.0 license (found in the
+ *  LICENSE file in the root directory of this source tree) and the GPLv2 (found
+ *  in the COPYING file in the root directory of this source tree).
+ *  You may select, at your option, one of the above-listed licenses.
+ */
+
+#pragma once
+
+#include <atomic>
```

Comment:
Same thing here, could you cleanup the include orderings?

Reply:
Did I reorder these includes right?